other-docs/guides/getting-help-with-altis.md

```diff
@@ -8,72 +8,66 @@ Support happens via the Altis Dashboard. From your Altis Dashboard you will be a
 
-You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.
+From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+If the site is down for example, select 'Report an Incident'.
 
-**Subject**
+**Report an ongoing issue**
 
-Short description of the support request.
+Issues, also known as 'Problems' are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, or if you've run into a bug, select 'Report an ongoing issue'.
 
-**Description**
 
-The body of the support request.
+**Request an infrastucture change**
 
-**CCs**
+There’s an array of tasks that you might need help with. Typically, this might be the provisioning of an additional environemnent, or it could be to run an operation that requires a cloud engineer, upgrading PHP for example.
 
-You can CC additional relevant parties to the support request if desired. They will recieve carbon copies of the correspondance.
+Tasks should be well described with detailed requirements and time scales. We cannot guarantee will not be able to service same-day requests, if there's capacity to do so, we will.
 
-## Support Ticket Types
+**Ask a question**
 
-We break down support into different categories. Each has its own language and processes, but it should be somewhat intuitive. Let's go over the types.
+Have a question for us? Ask!
 
-![Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu](./assets/altis-support-type.png)
+If you have a technical question or require clarification on a feature, just open a support question and our support team will do their best to answer you.
 
-We currently have four types of support. Below is a description of each and how they are different from each other.
+### Filling the form
 
-### Incident
+**Application**
 
-Incidents are unplanned events that negatively impact service, or the quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
+Select the specific Application or select all Application(s)/Environment(s) to which the support ticket concerns.
 
-From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
-If the site is down for example, select the 'Incident' type, with a priority of ‘Urgent’.
+**Priority**
 
-### Problem
+You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.
```

Comment:
Should we describe the intended use of priorities or no? e.g. urgent should only be used for site outage or serious problem happening right now. If you have other support requests in the queue then you could set questions or newer requests to a lower priority for example.

Something along those lines? Not a requirement though.

Reply:
That’s a good idea actually. I hadn’t thought of it like that. Commit incoming!

Reply:
I took a bash at guiding use for priorities.